PWGDQ/TableProducer/tableMaker_withAssoc.cxx

```diff
@@ -861,6 +863,56 @@ struct TableMaker {
           VarManager::FillZDC(bc_zdc);
         }
       }
+      // Fill FIT info using newbc pattern for UPC events (similar to ZDC)
+      if constexpr ((TEventFillMap & VarManager::ObjTypes::ReducedFit) > 0) {
+        if constexpr ((TEventFillMap & VarManager::ObjTypes::RapidityGapFilter) > 0) {
+          // The DQRapidityGapFilter contains the index of the bc we want to get FIT info from
+          auto newbc = bcs.rawIteratorAt(collision.newBcIndex());
+          // Fill FIT information from newbc
+          if (newbc.has_foundFT0()) {
```

Comment:
I think for all this block of code you can just call the VarManager::FillFIT(), otherwise, this VarManager function does not seem to be used anywhere